example_site/src/components/ConfigurationControls.js

```diff
@@ -282,61 +408,11 @@ export default function ConfigurationControls({ isOpen }) {
             Show the unsaved changes modal?
           </label>
 
-          <label
-            className="form-label configuration-column-3"
-            htmlFor="unsavedChangesModalTitle"
-          >
-            Modal Title
-          </label>
-          <input
-            type="text"
-            className="form-input"
-            name="unsavedChangesModalTitle"
-            defaultValue="Exit Editor"
-            ref={register}
-          />
-
-          <label
-            className="form-label configuration-column-3"
-            htmlFor="unsavedChangesModalBody"
-          >
-            Modal Body Text
-          </label>
-          <input
-            type="text"
-            className="form-input"
-            name="unsavedChangesModalBody"
-            defaultValue="Your video has unsaved edits. Are you sure you want to leave?"
-            ref={register}
-          />
-
-          <label
-            className="form-label configuration-column-3"
-            htmlFor="unsavedChangesModalConfirmButton"
-          >
-            Modal Confirmation Button Label
-          </label>
-          <input
-            type="text"
-            className="form-input"
-            name="unsavedChangesModalConfirmButton"
-            defaultValue="Exit Editor"
-            ref={register}
-          />
+          { 
+            //Collapse Unsaved Changes Confirmation Modal
+            shouldShowUnsavedChangesModal && renderUnsavedChanges() 
+          }
```

Comment:
I know they use this format in the React docs, but check out [this article](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx) from Kent Dodds about the dangers of using `&&` instead of ternaries while rendering. The basic idea is that when you use a ternary, you get you be explicit about what you want to render when your variable evaluates to `falsey`, and gives some more details about what React is actually doing when it evaluates these statements.

Reply:
Additionally, you can conditionally render the HTML here instead of defining a function above, like so:
```
{shouldShowUnsavedChangesModal ? (
    <input>....</input>
) : (null }
```